cl/search/models.py

```diff
@@ -613,7 +613,12 @@ def save(self, *args, **kwargs):
             self.docket_number_core = make_docket_number_core(
                 self.docket_number
             )
-        if self.source in self.RECAP_SOURCES:
+
+        update_fields = kwargs.get("update_fields")
+        incremented_view_count = (
+            True if update_fields and "view_count" in update_fields else False
+        )
+        if self.source in self.RECAP_SOURCES and not incremented_view_count:
```

Comment:
This is a funny way to fix this, no? What if we want to update a different field or just do a regular old save() on a docket with a source in RECAP_SOURCES? Wouldn't it make more sense to relax the validation error on Appellate court RECAP_SOURCE'ed dockets?

Reply:
Yeah, now we know that we might continue adding appellate dockets without `pacer_case_id` via RSS feeds, it makes more sense to just relax the validation for Appellate RECAP Dockets, changed it.